share/github-backup-utils/ghe-gc-disable

```diff
@@ -9,8 +9,8 @@
 set -e
 
 # Bring in the backup configuration
-cd $(dirname "$0")/../..
-. share/github-backup-utils/ghe-backup-config
+export GHE_BACKUP_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
```

Comment:
That is, reverting https://github.com/github/backup-utils/pull/179/commits/3427dfe7df2867bd54250b37ee7c4ae7e150efef.